### ISSUE TITLE
x86_64-config-4.14 does not work with 5.0-rc2 kernel anymore

### ISSUE THREAD

**Issue by atrosinenko** [tytso/xfstests-bld#18]
The latest sample kernel config in `kernel-configs` for now is for the 4.14 kernel on x86_64. It worked fine for launching the 4.20 kernel version, but somewhere between `4.20` - `5.0-rc2` it stopped working.

## How to reproduce

Checkout the `v4.20` tag, copy `x86_64-config-4.14` to `.config`, run `make olddefconfig`, then build and run -- it works fine.

Then repeat the same steps with the `v5.0-rc2` tag -- it panics because of absence of the root device:
```
[    1.042661] VFS: Cannot open root device "vda" or unknown-block(0,0): error -6
[    1.043315] Please append a correct "root=" boot option; here are the available partitions:
[    1.044047] Kernel panic - not syncing: VFS: Unable to mount root fs on unknown-block(0,0)
[    1.044766] CPU: 0 PID: 1 Comm: swapper/0 Not tainted 5.0.0-rc2-xfstests #1
[    1.045378] Hardware name: QEMU Standard PC (i440FX + PIIX, 1996), BIOS 1.11.1-1ubuntu1 04/01/2014
[    1.046155] Call Trace:
[    1.046381]  dump_stack+0x67/0x90
[    1.046677]  panic+0x100/0x2b3
[    1.046952]  mount_block_root+0x214/0x2be
[    1.047308]  ? do_early_param+0x8e/0x8e
[    1.047649]  prepare_namespace+0x130/0x166
[    1.048012]  kernel_init_freeable+0x316/0x325
[    1.048397]  ? rest_init+0x24c/0x24c
[    1.048715]  kernel_init+0xa/0x104
[    1.049018]  ret_from_fork+0x3a/0x50
[    1.049420] Kernel Offset: 0xcc00000 from 0xffffffff81000000 (relocation range: 0xffffffff80000000-0xffffffffbfffffff)
[    1.050347] Rebooting in 5 seconds..
```

I use Ubuntu 18.10 and its packaged version of QEMU
```
$ qemu-system-x86_64 --version
QEMU emulator version 2.12.0 (Debian 1:2.12+dfsg-3ubuntu8.2)
Copyright (c) 2003-2017 Fabrice Bellard and the QEMU Project developers
```

[dmesg-v5.0-rc2.log](https://github.com/tytso/xfstests-bld/files/2755259/dmesg-v5.0-rc2.log)

**Comment by tytso** [comment]
This was fixed as of commit 4fc9e3689043c74a89db55c36c8df2dbe9703026

**Comment by atrosinenko** [comment]
Thank you!
